### PR TITLE
Merge gif-adapta into master to add GIF version of Adapta theme

### PR DESCRIPTION
**Pull request by RedFantom**
I have forgotten to add the GIF version of the adapta theme to the package. This is probably also the cause of #78 .

This branch contains the GIF version of the adapta theme, albeit not without a few GIF-artifacts, because Adapta uses some transparency.

### CODE REVIEW

File: tests/test_widgets.py [hunk 1]
```diff
@@ -49,6 +49,11 @@ def test_widget_creation(self):
         if "signal" not in locals() or not hasattr(signal, "alarm"):
             return
         for theme in self.window.themes:
+            # Delete all existing theme images to reduce GPU memory usage
```
Comment (Akuli) [inline tests/test_widgets.py:52]: To me, this seems to be a workaround for a more serious underlying issue. What code creates these images? Why is that code not deleting them?

Comment (RedFantom) [reply tests/test_widgets.py:52]: You make a good point, one that I actually also started thinking about. The images are created by loading the theme, but there's currently no way to distinguish them from user-created images, which the code should not delete.

Basically, if you load all themes all images will be stored on GPU memory, as far as I understand it. I have seen this issue crop up on AppVeyor before somewhere, and I believe it was due to the 8MiB limit (or something similarly low). I don't remember exactly where this was and how I fixed it, though, so I was hoping that simply trying to delete the images would be sufficient (they at least get deleted when running the tests locally).

So really what's needed is:
- [ ] A way to distinguish theme images from user images (e.g. by specifying a name on creation, like `adapta1`, `adapta2`, etc.
- [ ] Method for deleting the images that actually works
- [ ] Code that deletes the images upon theme switch

Comment (Akuli) [reply tests/test_widgets.py:52]: As a side note, not all images are "theme images" or "user images", at least on my system:

```
>>> import tkinter
>>> root = tkinter.Tk()
>>> root.eval('image names')
'::tk::icons::information ::tk::icons::error ::tk::icons::warning ::tk::icons::question'
```

These images seem to be used in `tkinter.messagebox` dialogs.

Comment (RedFantom) [reply tests/test_widgets.py]: So, after doing some more research, the following is what I found:

The last time I saw this issue, it was in a program that also loaded many PhotoImages. There, I was able to change it so that the images were deleted, and then the memory problems were fixed. However, for ttk themes, there is no unloading mechanism. Tcl apparently lacks unloading mechanisms for most things, including packages, and even if a package specifies one, interpreters that have 'safe' settings will never use them because they're too dangerous.

This is because of the way the theme works. When a photo image is 'deleted' with the command, its deletion is only scheduled for after all references to the image have been removed. And since ttk style elements cannot be changed or deleted after they have been created and the theme they are a part of cannot be unloaded, the reference to the image is permanent for program runtime, and deletion never happens.

So, that leaves the current issue. AppVeyor has a serious issue with loading more than about 800 photo images into memory, so not all themes can be loaded for a single Tk instance on AppVeyor. As for normal usage, the conclusion is that switching themes will cause a leak in memory of a couple hundred KiB per theme that cannot be prevented. At least, not without either changing the Tcl source code or maybe using some weird memory hack in C code, neither of which I'm willing to resort to.

But it gets worse. This issue also occurs on my Windows 7 VM (with 256MiB of VRAM), so I suppose there is some other memory limitation than simply the full VRAM. There is just one theme too many. On another note, I changed the wrong test before: The test I changed doesn't even run on Windows, so I fixed that mistake.

**TL;DR;** Due to Tcl/Tk limitations, the number of themes that are loaded should be limited to the minimum required as each theme that was at some point loaded and then no longer used remains in memory with no the possibility of throwing it out.
